ledger/participant-integration-api/src/main/scala/db/migration/translation/TransactionSerializer.scala

```diff
@@ -50,7 +49,7 @@ private[migration] object TransactionSerializer extends TransactionSerializer {
             TransactionCoder.EventIdDecoder(trId),
             ValueCoder.CidDecoder,
             TransactionOuterClass.Transaction
-              .parseFrom(Reader.damlLfCodedInputStream(stream)),
+              .parseFrom(com.google.protobuf.CodedInputStream.newInstance(stream)),
```

Comment:
If values have a recursion limit of 100, shouldn't transactions have a higher (non-default) limit? 104 maybe?